tests/scripts/all.sh

```diff
@@ -6396,6 +6396,18 @@ component_check_test_helpers () {
     python3 -m unittest tests/scripts/translate_ciphers.py 2>&1
 }
 
+component_test_tls13_only () {
+    msg "build: default config without MBEDTLS_SSL_PROTO_TLS1_2"
+    scripts/config.py set MBEDTLS_SSL_EARLY_DATA
+    scripts/config.py set MBEDTLS_SSL_RECORD_SIZE_LIMIT
+}
```

Comment:
This doesn't seem correct - there should already be a component called this.

Reply:
> This doesn't seem correct - there should already be a component called this.

That moment you didn't stage the deletion of that hunk you copied as a template :-p

Reply:
I suspected something like that!